collect/collect.go

```diff
@@ -1737,3 +1734,19 @@ func (i *InMemCollector) sendDecisions(decisionChan <-chan TraceDecision, interv
 		}
 	}
 }
+
+func (i *InMemCollector) recordQueueMetrics() {
+	ticker := time.NewTicker(time.Second)
```

Comment:
One second means your histograms are gonna be really sparse. I guess we can try it and see how it goes but this is cheap enough that using say 100ms would probably have no impact on load and be more likely to capture volatility in the queue lengths.

Reply:
I changed it to be 100ms